Articles/Defining and Using Observables.md

```diff
@@ -0,0 +1,341 @@
+
+Example in a `dictionary.json` file:
+```jsonc
+{
+  "name": "dictionary",
+  // ... other dictionary fields (caption, description, types)
+  "attributes": {
+    // ... other attributes
+    "cmd_line": {
+      "caption": "Command Line",
+      // ... other attribute fields
+
+      // This is the observable definition for the cmd_line attribute
+      "observable": 13
+    },
+    // ...
+  },
+  // ...
+}
+```
+
+### Definition Example: Observable by Object
+Defining an observable by object is in a metaschema object definition file. The definition is done by adding an `observable` field directly in the object definition object. This also works for object definitions that extend another, including the special patch case. For both regular and patch extends cases, the observable definition adds or replaces any existing object-level observable definition.
+
+This, along with defining observables by dictionary type, are the broadest ways to define observables. All instances attributes of this type (regardless of attribute name) become observables.
+
+Example in object definition file `objects/container.json`:
+```jsonc
+{
+  "name": "container",
+
+  // This is the observable definition for this object
+  "observable": 27,
+
+  // ... other object definition fields
+}
+```
+
+### Definition Example: Observable by Event Class Attribute / Observable by Object Attribute
+Observables can be defined in event class or object attributes. These are very similar and so are described together here. Defining observables this way limits the scope where these observables will occur. Note that definitions of this type are for attributes directly defined in the event class or object, not for attributes in a nested structure.
+
+Defining observables in event class or object attributes works just as with any other attribute definition at these levels: the fields defined override the same field defined in the dictionary or any event classes / objects the current item is derived from.
+
+Definitions of this type work for event class and object `extends` definitions both for the normal case (subclass / subtype) as well as the "patch extends" case, however "hidden" event classes and objects are not supported due to the potential of colliding observable `type_id` values. See [Appendix 2: Hidden Types](#appendix-2-hidden-types) for more about this issue.
+
+Example in object definition file `objects/cve.json`:
+```jsonc
+{
+  "name": "cve",
+  // ... other object definition fields
+  "attributes": {
+    // ... other attributes
+    "uid": {
+      "caption": "CVE ID",
+      // ... other attribute fields
+
+      // This defines uid as an observable when directly inside a cve object
+      "observable": 18
+    },
+    // ...
+  },
+  // ...
+}
+```
+
+Event classes work identically: attribute observable definitions occur in attribute detail objects inside the `attributes` object.
```

Comment:
This isn't clear: I wouldn't use the term 'object' here (JSON object) since it could be confused with OCSF objects used up until now.  Maybe say stanza, or section and subsection?